packages/terra-navigation-side-menu/src/_MenuItem.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/role-supports-aria-props */
```

Comment:
This seems shady too.

Reply:
Also addressed here: https://github.com/cerner/terra-framework/pull/1164